_sass/partials/_typography.scss

```diff
@@ -229,6 +229,7 @@ h5 {
 a {
   color: $brand-red;
   text-decoration: none;
+  font-weight: 600;
```

Comment:
Font is accessible in 16px when font weight is 600:
<img width="709" alt="Screenshot 2023-06-28 at 13 27 52" src="https://github.com/solita/solita.github.com/assets/52316685/d45d9ac3-8a01-463e-9312-e8a7d4391d16">